forge-gui/res/cardsfolder/upcoming/gríma_sarumans_footman.txt

```diff
@@ -0,0 +1,11 @@
+Name:Gríma, Saruman's Footman
```

Comment:
This needs to say Grima not Gríma (we can't use accents for this)
Also important here, the card file name itself should say grima not gríma